QandA/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+# Create your models here.
+class user(models.Model):
+    user_id = models.IntegerField(primary_key=True)
+    password = models.CharField(max_length=50)
+    user_name = models.CharField(max_length=100)
+
+class meetings(models.Model):
+    meeting_id = models.IntegerField(primary_key=True)
+    venue = models.CharField(max_length=50,)
+    moderator_id = models.IntegerField()
```

Comment:
Shouldn't this be a `ForeignKey` https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.ForeignKey ?

Reply:
Changed it to foreign key to Django User model.